api/helpers/create-csv-data-dump.js

```diff
@@ -0,0 +1,63 @@
+const fs = require("fs");
+const { parse } = require("json2csv");
+const {
+  db,
+  LIST_ARTICLES,
+  CASE_BY_ID,
+  METHOD_BY_ID,
+  ORGANIZATION_BY_ID
+} = require("./db.js");
+
+async function createCSVDataDump(type) {
+  const entries = await db.many(LIST_ARTICLES, {
+    type: type + "s",
+    lang: "en"
```

Comment:
don't hardcode language here. use locale cookie and fallback to "en"

Reply:
upon further testing, looks like we need to hardcode "en" here until we have populated the localized_texts table with machine translations for other languages.